mkt/api/urls.py

```diff
@@ -41,7 +41,8 @@ def include_version(version):
     return include('mkt.api.v%d.urls' % version, **kwargs)
 
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
+    '',
```

Comment:
I find the previous version more readable :(

I wonder if, here and in the various places where we call `patterns()` with `''` as the prefix, we actually need the function call at all. All `patterns()` does is add a prefix and wrap tuples of url+function in `url()` calls, if you are already using `url()` everywhere and you don't set a prefix, you could just do `urlpatterns = [...]`.